src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -31,9 +31,21 @@ public class WalletCreationRequest : RequestModel
     {
         public string Mnemonic { get; set; }
 
+        /// <summary>
+        /// When supplied this password is used to encrypt the wallet.
+        /// </summary>
+        /// <remarks>
+        /// If the account was created previously before passphrase was available, then the password was used as passphrase. It is therefor essential that
```

Comment:
This doc is more appropriate to RecoverWallet than to CreateWallet. Can you please update it?